src/styles/BookDetails.module.css

```diff
@@ -23,23 +23,33 @@ p {
   padding-left: 0;
 }
 
-.details ul li {
-  margin: 0 5px;
-}
-
 .details p {
   color: #4386bf;
 }
 
-.details a {
-  text-decoration: none;
+.line {
+  border-right: 3px solid #e9e9e9;
+  border-top: none;
+  border-bottom: none;
+  border-left: none;
+}
+
+.button {
+  border: none;
+}
```

Comment:
- [ ] Kindly it will be better 😃 if you increase the margin between the cards to make it like the original design as you see here it is less than the original design,s o let's increase the margin a little to make it like the original design 💯😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|  ![image](https://github.com/tumainimaganiko/react-bookstore/assets/83360636/77c55429-dd7e-4026-9cbd-e8bf16e70067)  |    ![image](https://github.com/tumainimaganiko/react-bookstore/assets/83360636/8eb87b57-a0b7-4518-947a-0c1bc0075ab0)  |

Reply:
Thank's for your suggestion, I have also noticed it and I have already fixed the issue 😃